src/demo/coupons.js

```diff
@@ -119,18 +119,17 @@ const getAllCodes = (couponHasError = false) => [...Array(codesCount)].map((_, i
 const allCodes = getAllCodes();
 
 const unassignedCodes = allCodes.filter(code => (
-  code.redemptions.available > minRedemptionsCount && code.redemptions.used < maxRedemptionsCount
+  code.assigned_to === undefined && code.redemptions.used < code.redemptions.available
 ));
 const unredeemedCodes = allCodes.filter(code => (
-  (code.redemptions.available === minRedemptionsCount &&
-    code.redemptions.used > minRedemptionsCount && code.redemptions.used < maxRedemptionsCount)
+  code.assigned_to !== undefined && code.redemptions.used < code.redemptions.available
```

Comment:
This filter is for unredeemed codes. The logic here should be `code.redemptions.used === 0`, otherwise you'll get codes that have been partially redeemed.

Reply:
Actually `used` key from backend actually calculates this based on a voucher's assignments + it's redemptions ... and `available` is `max_use` ... so if a voucher has 10 `max_use`, it has 10 potential seats for `used` column, when 1 code is assigned ( it means 1 assignment used / 10 available), we still get to get the voucher in UNASSIGNED filter because it still has 9 potential seats left ... 

We will get the voucher in unassigned filter unless it has no slot available left for assignment ...

Reply:
The unredeemed filter means it gets the coupon vouchers which have been assigned but have not yet been redeemed ...

However, that said, this should have a had assigned_to value set ... I will need to fix this.

Reply:
Fixed `assigned_to` value set if a voucher is assigned otherwise it is `undefined`